src/odil/Reader.cpp

```diff
@@ -469,7 +471,9 @@ Reader::Visitor
 ::operator()(Value::Binary & value) const
 {
     auto const vl = this->read_length();
-    if(vl == 0xffffffff)
+    if(vl == 0)
```

Comment:
Please avoid blocks without braces: the [goto fail](https://www.imperialviolet.org/2014/02/22/applebug.html) bug is still in many memories 😄